Sources/Sentry/include/SentryLog.h

```diff
@@ -1,51 +0,0 @@
-#import "SentryDefines.h"
```

Comment:
The impl of this header file was done in `SentryLogC.h`. So the declarations were moved to `SentryLogC.h` and this header removed to avoid naming conflicts.